src/core/ts/config.ts

```diff
@@ -69,6 +69,29 @@ export const modelRegistry: ModelRegistry = {
         "909b1eea1face0d7f90a474fe29a8c0fef8d104b6e41e65616f864c964ba8845",
     },
   },
+  csen: {
```

Comment:
is it generated by some script? where is it? ideally it should be a part of CI

Reply:
At this point, with the extremely low amount of languages supported and model updates, I have chosen the "don't spend an hour automating something that takes five minutes to do by hand" path. I agree that a script would be good as an enhancement, not sure if it should be part of this repo or the bergamot-models repo for instance since ideally the registry should not be bundled in the extension in the future, instead remotely updated.

Reply:
Filed: https://github.com/mozilla-applied-ml/bergamot-models/issues/16

Reply:
Transferred issue to https://github.com/mozilla-applied-ml/bergamot-model-registry/issues/1 so that it is not dependent on the long term survival of the bergamot-models repo.